Frameworks/CoreText/CTRun.mm

```diff
@@ -256,11 +261,15 @@ void CTRunDraw(CTRunRef run, CGContextRef ctx, CFRange textRange) {
         range.length = [string length];
     }
 
-    int numGlyphs = curRun->_characters.size();
+    int numGlyphs = curRun->_glyphAdvances.size();
     WORD* glyphs = (WORD*)IwMalloc(sizeof(WORD) * numGlyphs);
 
+    WORD* characters = (WORD*)IwMalloc(sizeof(WORD) * numGlyphs);
```

Comment:
This code is going to be thrown away so kept with the existing coding style

---
In reply to: [77862776](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77862776) [](ancestors = 77862776,77747600,77732196)